crypto_api_spongycastle/build.gradle

```diff
@@ -0,0 +1,4 @@
+dependencies {
+    implementation "com.madgag.spongycastle:prov:$spongycastleVersion"
```

Comment:
perhaps we should also define spongycastleVersion in this module as it is only used here *YAY*